swagger.yaml

```
@@ -1,6 +1,6 @@
 openapi: 3.0.0
 info:
-  version: "0.1.24"
+  version: "0.1.25"
```

```suggestion
  version: "0.1.24"
```

Thanks a million for the PR, good catch! Let's keep the version as is and we'll bump it with the next release. Only `released` branch is getting published, so we don't have to modify it in master.

Thanks once again!